### ISSUE TITLE
chore: add `Français - fr-FR` language support

### ISSUE THREAD

**Issue by brionmario** [asgardeo/javascript#156]
chore: add `Français - fr-FR` language support

### Current Limitation

Currently, the Asgardeo JavaScript SDK does not have support for **French (Français - fr-FR)**. Users who prefer French will not see UI components translated, limiting accessibility for French-speaking users.

> [!Important]
> Contributors for this task should be **native French speakers or have professional fluency** to ensure accurate, culturally appropriate translations.

### Suggested Improvement

Add **French (`fr-FR`) language support** to the `@asgardeo/i18n` package.

**Implementation Plan:**

1. Create a new locale file:

    Follow https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md#contributing-to-the-internalization-i18n-package

3. Add documentation for usage:  
   - Update the SDK docs: https://wso2.com/asgardeo/docs/sdks/react/overview/

### Please select the package issue is related to

@asgardeo/react, @asgardeo/nextjs

### Version

current

### Reporter Checklist

- [x] I have searched the existing issues and this is not a duplicate.
- [x] I have provided all the necessary information.
- [x] I have verified the improvement is not available in the latest version of the package.

**Comment by brionmario** [comment]
Hey @lemeurherve,

Since you've shown your interest in https://github.com/asgardeo/javascript/issues/145#issuecomment-3337569701.

I will go head and assign you.

Please have a look at our [CONTRIBUTION](https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md) guideline before proceeding with the solution.

Let's discuss here in case if you need any clarifications.

Good Luck ! 🙌

**Comment by lemeurherve** [comment]
Thanks @brionmario 

Where the list of const to be translated can be found?

**Comment by lemeurherve** [comment]
Opened https://github.com/asgardeo/javascript/pull/188
